packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -256,18 +260,33 @@ export function useBlocksStore() {
          */
         const [createdNode] = Editor.parent(editor, editor.selection.anchor.path);
```

Comment:
`createdNode` name signifies its a brand new node created, rather its just an existing node splitted, we can call it fragmentedNode or something like that